.eslintrc.js

```diff
@@ -114,6 +114,18 @@ module.exports = {
         'import/no-extraneous-dependencies': 0,
       },
     },
+    {
+      files: ['test-types/*.test.ts'],
+      rules: {
+        '@typescript-eslint/array-type': 0,
+        'eslint-comments/disable-enable-pair': 0,
+        'import/no-extraneous-dependencies': 0,
+        'jest/no-focused-tests': 0,
+        'jest/no-identical-title': 0,
+        'jest/valid-expect': 0,
+        'sort-keys': 0,
```

Comment:
```suggestion
        'jest/no-focused-tests': 0,
        'jest/no-identical-title': 0,
        'jest/valid-expect': 0,
```

the other rules should be adhered to, imo. `import/no-extraneous-dependencies` should be fixed via allowing it as dev dep here: https://github.com/facebook/jest/blob/c9c8dba4dd8de34269bdb971173659399bcbfd55/.eslintrc.js#L130-L137

Reply:
Doesn't work @SimenB. It then shows that `@jest/globals` has to be installed. But other than that, dev dep error gets fixed.

```
'@jest/globals' should be listed in the project's dependencies. Run 'npm i -S @jest/globals' to add it
```

Reply:
add an eslint ignore comment on that specific line

Reply:
Sure, I've did that :)